src/main/java/com/epam/lab/payments/services/UserDetailsServiceImpl.java

```diff
@@ -12,25 +12,21 @@
 import org.springframework.stereotype.Service;
 import org.springframework.transaction.annotation.Transactional;
 
-import java.util.Arrays;
 import java.util.HashSet;
-import java.util.List;
 import java.util.Set;
 
 @Service(value = "userDetailService")
 @RequiredArgsConstructor
 public class UserDetailsServiceImpl implements UserDetailsService {
     private final UserRepository userRepository;
-
-    private List getAuthority() {
-        return Arrays.asList(new SimpleGrantedAuthority("ROLE_USER"));
-    }
-
+    
     @Override
     @Transactional(readOnly = true)
     public UserDetails loadUserByUsername(String userEmail) throws UsernameNotFoundException {
```

Comment:
`UsernameNotFoundException` is runtime exception so it is unnecessary to write it in method declaration.

Reply:
done